### ISSUE TITLE
OpenEmu won't respond at all (v2.2.1) (OS ver. 10.15.6)

### ISSUE THREAD

**Issue by coolman6** [OpenEmu/OpenEmu#4299]
Just recently, OpenEmu stopped responding for me. I don't know why, but it just stopped working. I'm using the latest version, 2.2.1, on the latest OS on Mac, 10.15.6. Every time I try to load it, it opens, but immediately starts doing the "beach ball" cursor thing for a long, long time and I'm unable to click anything. I'm only able to exit via force quitting. I tried deleting it and reinstalling it but that didn't work. I tried shutting down the computer and turning it back on and still no luck. Is there a way to fix this because I'm really bummed out right now and I really don't want to quit using OpenEmu since I enjoy it so much.

**Comment by niccolomineo** [comment]
I am using 2.2.1 on 10.15.6 without any problems, so you should try and remove the app completely via CleanMyMac or CCleaner. Also, before reinstalling it, try backing up your games, deleting OpenEmu's game folder and readding the games.

**Comment by kable-wilmoth** [comment]
I have never tried OpenEmu before but was going to look into it to recommend to a friend.  I downloaded v2.2.1 and dropped into my mac's application folder.  After allowing the app to run (security wise) the app doesn't display or do anything.  It is in the system tray and just says app is not responding.  My mac is 10.15.6

This is the output if ran from the terminal.

/Applications/OpenEmu.app/Contents/MacOS
❯ ./OpenEmu
objc[45981]: Class IKCGRenderer is implemented in both /System/Library/Frameworks/Quartz.framework/Versions/A/Frameworks/ImageKit.framework/Versions/A/ImageKit (0x7fff868e8660) and /Applications/OpenEmu.app/Contents/MacOS/./OpenEmu (0x103257860). One of the two will be used. Which one is undefined.
2020-09-26 16:26:11.899 OpenEmu[45981:1394168] OELibraryDatabase loadFromURL: 'file:///Users/<user>/Library/Application%20Support/OpenEmu/Game%20Library/'
CoreData: annotation:  Failed to load optimized model at path '/Applications/OpenEmu.app/Contents/Resources/OEDatabase.momd/OEDatabase 1.3.omo'

**Comment by Reeywhaar** [comment]
...sorry for possible offtop

@kable-wilmoth Just wanted to say thank you very much! I'm now able to play again! My problem was that after upgrade to Catalina OpenEmu wasn't able to open any game. Just nothing did happen upon click. I've tried to watch logs, but there wasn't something my eye can catch on. And I haven't guessed to open OpenEmu via terminal. And after your message I've tried, and that is what I got.

```
objc[8259]: Class IKCGRenderer is implemented in both /System/Library/Frameworks/Quartz.framework/Versions/A/Frameworks/ImageKit.framework/Versions/A/ImageKit (0x7fff90ef0660) and /Applications/OpenEmu.app/Contents/MacOS/OpenEmu (0x1014ed860). One of the two will be used. Which one is undefined.
2020-09-29 22:16:56.956 OpenEmu[8259:129876] OELibraryDatabase loadFromURL: 'file:///Users/reeywhaar/Library/Application%20Support/OpenEmu/Game%20Library/'
CoreData: annotation:  Failed to load optimized model at path '/Applications/OpenEmu.app/Contents/Resources/OEDatabase.momd/OEDatabase 1.3.omo'
2020-09-29 22:16:57.181 OpenEmu[8259:129876] window did load
2020-09-29 22:16:57.588 OpenEmu[8259:129876] OEHandle_DeviceMatchingCallback: failed to open device at 0x7fbd8b48d1f0

{HERE I CLICK ON A GAME}
/Users/reeywhaar/Library/Application Support/org.openemu.OEXPCCAgent.Agents/org.openemu.OEXPCCAgent.2B22EC59-0BD0-40C9-A99D-089FC783B9C4.plist: No such file or directory
```

So, after I removed `~/Library/Application Support/org.openemu.OEXPCCAgent.Agents` OpenEmu started to work properly.
Thank you! 👍

By the way, it seems our both logs are similar. So, at least failing on not being able to read `OEDatabase 1.3.omo` is not the case, though I see that file is present.

**Comment by clobber** [comment]
This permissions problem is resolved in OpenEmu 2.3; we don't use that folder anymore.
